README.md

```diff
@@ -1,42 +1,62 @@
 # Flutter Templates
 
-All the templates can be used to kick off a new Flutter application quickly.
+All the templates can be used to kick off a new Flutter project quickly.
 
-## Usage
+## Features
 
-Clone the repository
+- Supports __Android__ and __iOS__ platforms *(Web and Desktop are not yet supported)*.
```

Comment:
@doannimble IMO `are not yet supported` is more correct here. I want to share the project status instead of the action progress as we did not start on that support. More at https://ell.stackexchange.com/questions/69482/difference-between-are-not-yet-and-have-not-yet